src/core/persistent_backend_manager.cc

```diff
@@ -61,7 +61,8 @@ Status
 PersistentBackendManager::InitPersistentBackends(
     const BackendCmdlineConfigMap& config_map)
 {
-  for (const auto& be : {"pytorch", "tensorflow", "onnxruntime"}) {
+  // FIXME this should be customized based on which backend is enabled.
+  for (const auto& be : {"pytorch", "tensorflow", "onnxruntime", "custom"}) {
```

Comment:
We shouldn't need to persistently load the "custom" backend so it doesn't need to be added here.

Reply:
Yes, removed.